pkg/windows/windows.go

```diff
@@ -234,8 +234,8 @@ type Windows interface {
 	Bootstrap(string, string, string) error
 	// ConfigureWICD ensures that the Windows Instance Config Daemon is running on the node
 	ConfigureWICD(string, string) error
-	// Deconfigure removes all files and networks created by WMCO and runs the WICD cleanup command.
-	Deconfigure(string, string) error
+	// Deconfigure removes all files and networks created by WMCO
+	Deconfigure() error
```

Comment:
This isn't really `Deconfigure` at this point. Its no longer undoing changes done by `ConfigureWICD`
It may be more apt to name it `RemoveFilesAndNetworks`

Reply:
I agree with this given that we no longer have `Windows.Configure()`. In some ways you are doing the opposite of `windows.Bootstrap()`.